consai_game/consai_game/play/conditions/robot_conditions.py

```diff
@@ -0,0 +1,19 @@
+from consai_game.world_model.world_model import WorldModel
+from consai_game.core.play.play_condition import PlayCondition
+
+
+class RobotConditions:
+    """ロボットに関連する条件をまとめたクラス."""
+
+    @staticmethod
+    def their_robots_in_our_area() -> PlayCondition:
```

Comment:
判定条件もわかるような名前にしたいです。
例えば `half` をいれるとよいかな

```suggestion
    def are_half_of_their_robots_in_our_area() -> PlayCondition:
```